.github/workflows/ci.yml

```
@@ -147,6 +147,30 @@ jobs:
       - name: Test generated standalone binary
         run: yarn dist-standalone:test
 
+  standalone-binary-test-ubuntu-arm:
+    name: Test standalone binary in ARM ubuntu
+    runs-on:
+      group: ARM LINUX SHARED
+      labels: arm-4core-linux
```

Apparently, [other Datadog repos](https://github.com/search?q=org%3ADataDog+%22arm-4core-linux%22&type=code) use the following syntax:
```suggestion
    runs-on: arm-4core-linux
```

---

I'm using this syntax as it's the one the folks at #build-support recommended ([docs](https://datadoghq.atlassian.net/wiki/spaces/DEVX/pages/3790274750/Github+Actions)), in theory this should be more robust against changes

---

Thanks, I wasn't aware of these docs 👍